addons/packages/ako-operator/1.5.0/README.md

```diff
@@ -1,22 +1,96 @@
-# ako-operator
+# Load Balancer Operator
 
-This package provides NSX Advanced Load Balancer using ako-operator.
+This package reconciles Cluster API objects and manages the lifecycles of load balancers using ako-operator.
```

Comment:
I suggest to specify `AVI load balancers` instead of generic `load balancers` to make it clear the operator is specific for AVI.

Reply:
Good point!